etna/search/forms.py

```diff
@@ -198,20 +237,238 @@ class BaseCollectionSearchForm(forms.Form):
         required=False,
     )
 
+    def _get_compressed_date_or_none_on_error(
+        self, start_day: str, start_month: str, start_year: str
```

Comment:
There doesn't appear to be anything about this method that is specific to 'start' dates VS 'end' dates or some other date, so maybe less verbose variable names would be appropriate here?
```suggestion
        self, day: str, month: str, year: str
```